circleci/circleci.sh

```diff
@@ -0,0 +1,21 @@
+DEFAULT_LANG="ruby"
+lang=$DEFAULT_LANG
+
+detectProjectLang () {
+    if [ -f package.json ]; then
+        lang="nodejs"
+    fi
+    if [ -f Rakefile ]; then
+        lang="ruby"
+    fi
+    if [[ -n $(find . -maxdepth 1 -type f -name "*.go") ]]; then
+        lang="go"
+    fi
+    if [[ -n $(find . -name "__init__.py") ]]; then
+        lang="python"
+    fi
+}
+detectProjectLang
+
+BasePath=$1
+cp $BasePath/config-$lang.yml ./.circleci/config.yml
```

Comment:
and we need to change this to oya render. here and in docker pack. We can do it together after standup